subjects/0-shell/README.md

```diff
@@ -0,0 +1,81 @@
+## 0-shell
+
+### Objective
+
+The objective of this project is for you to create a simple [shell](https://en.wikipedia.org/wiki/Shell_script).
+
+Through the `0-shell` you will get to the core of the `Unix` system and explore an important part of this system’s API witch is the process creation and synchronization.
+Executing a command inside a shell implies creating a new process, which execution and final state will be monitored by its parents process. This set of functions will be the key to success for your project.
+
+For this project you will only have to create a simple `Unix` interpreter where you can run some of the most known commands. For this part of the project, no advanced functions, pipes or redirection will be asked, you can add them if you like.
```

Comment:
"shell", "Unix shell" or "command-line interpreter" but not "Unix interpreter". Unix can't be interpreted